capstoneteamb/capstone360
# View Student Page Changes (I would like to get this checked in today)

Copied from previous pull attempt and modified slightly:

How to test:
-For the student page:
With no reviews filled out, click any student's name in the professor dashboard. You will be forwarded to the students page. Once there, you should see a list of the names of all the student's team members on both the left (midterm) and right (final) sides of the page (the names are clickable buttons that allow you to see the reviews the student filled out for those team members). No reviews have been submitted at this point though, so all of the names should be italicized, have " - not completed" next to their names, and clicking on them should not redirect you anywhere. This is what the students page will look like when no reviews are filled out. There are also "Reports" links at the bottom of the two boxes (sides) of the page. These don't do anything at the moment. If something goes wrong loading the page, you should see a "something went wrong" error message displayed, though I don't expect it to.
If you do fill out a review by navigating to the /review page, click on the student's name in the dashboard as before and you will see that the user names on the left (midterm) side of the page are not italicized and don't have " - not completed" next to them. Clicking them should take you to the reviews page, which displays the review for the team member whose name you clicked on. If something goes wrong loading the page, you will see a "something went wrong" error message (like in the student page), though again, I don't expect it to.

Expected inputs: none (my code relies on what is in the database, not exactly from user input)

Expected Outputs: none (what it displays/doesn't display depends on what we find in the database)

Edge cases: I tried to handle edge cases (missing data/errors) in my code (hopefully I didn't miss any). I would think it should work in general.

Success cases: General use

Failure cases: None perhaps, that are very likely

No automated tests yet
